builds/e2e/templates/deploy-iotuap.yaml

```diff
@@ -109,6 +109,7 @@ steps:
         $testInfo=$testInfo+",ImagesSourceBranch=${{ parameters['images.source.branch'] }}"
         $testInfo=$testInfo+",HostPlatform=${{ parameters['metricsCollector.hostPlatform'] }}"
         $testInfo=$testInfo+",TestName=${{ parameters['testName'] }}"
+        $testInfo=$testInfo+",UpstreamProtocol=amqp"
```

Comment:
WHy do we need this?

Reply:
TestInfo is now used for both metrics upload and test result reporting. I don't beleive it is currently used for metrics, but since the upstream protocol is configurable then we should know about the setting in the test report.